src/OrdersList/hooks/useOrders/useOrders.test.js

```diff
@@ -0,0 +1,108 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { useLocation } from 'react-router';
+import { QueryClient, QueryClientProvider } from 'react-query';
+
+import { useOkapiKy } from '@folio/stripes/core';
+
+import { useBuildQuery } from '../useBuildQuery';
+import { useOrders } from './useOrders';
+import { order } from '../../../../test/jest/fixtures';
+
+jest.mock('react-router', () => ({
+  ...jest.requireActual('react-router'),
+  useLocation: jest.fn(),
+}));
+jest.mock('@folio/stripes/core', () => ({
+  ...jest.requireActual('@folio/stripes/core'),
+  useNamespace: () => ['namespace'],
+  useOkapiKy: jest.fn(),
+}));
+jest.mock('../useBuildQuery', () => ({ useBuildQuery: jest.fn() }));
+
+const orders = [order];
+const queryMock = '(cql.allRecords=1) sortby metadata.updatedDate/sort.descending';
+
+const queryClient = new QueryClient();
+// eslint-disable-next-line react/prop-types
+const wrapper = ({ children }) => (
+  <QueryClientProvider client={queryClient}>
+    {children}
+  </QueryClientProvider>
+);
+
+describe('useOrders', () => {
+  beforeEach(() => {
+    useBuildQuery.mockReturnValue(jest.fn(() => queryMock));
+
+    useOkapiKy
+      .mockClear()
+      .mockReturnValue({
+        get: () => ({
+          json: () => ({
+            purchaseOrders: orders,
+            totalRecords: orders.length,
+          }),
+        }),
+      });
+  });
+
+  it('should return an empty list if there no filters were passed in the query', async () => {
+    useLocation
+      .mockClear()
+      .mockReturnValue({ search: '' });
+
+    const { result, waitFor } = renderHook(() => useOrders({
+      pagination: { limit: 5, offset: 0, timestamp: 42 },
+    }), { wrapper });
+
+    await waitFor(() => !result.current.isFetching);
```

Comment:
as I understand, `isFetching` is not returned by the `useOrders` hook, maybe you mean `isLoading`?

Reply:
yep, it's copy paste